docs/book/src/reference/envtest.md

```diff
@@ -77,7 +77,17 @@ Logs from the test runs are prefixed with `test-env`.
 
 Controller-runtime’s [envtest][envtest] framework requires `kubectl`, `kube-apiserver`, and `etcd` binaries be present locally to simulate the API portions of a real cluster.
 
-For projects built with plugin v3+ (see your PROJECT file's `layout` key), the `make test` command will install these binaries to the `testbin/` directory and use them when running tests that use `envtest`.
+For projects built with plugin v3+ (see your PROJECT file's `layout` key), the `make test` command will install these binaries to the `bin/` directory and use them when running tests that use `envtest`.
+Ie,
+```shell             
+./bin/k8s/
+└── 1.24.2-darwin-amd64
+    ├── etcd
+    ├── kube-apiserver
+    └── kubectl
+
+1 directory, 3 files
```

Comment:
@rumstead that is the default behaviour so I think we need to put on top of the doc and we no longer need to say with the plugin go/v3.

Therefore, we need to recheck the current info in this doc since it seems outdatate.

Could you please help us with?
WDYT? is that make sense?

Reply:
certainly can help and makes sense